src/constants/sinsemilla.rs

```diff
@@ -161,27 +161,18 @@ mod tests {
     #[test]
     fn i2lebsp_k_round_trip() {
         {
-            let bitstring = (0..K).map(|_| rand::random()).collect::<Vec<_>>();
-            assert_eq!(
-                i2lebsp_k(lebs2ip_k(&bitstring) as usize).to_vec(),
-                bitstring
-            );
+            let bitstring = [0; K].map(|_| rand::random());
```

Comment:
You can map arrays now??? :exploding_head:

Reply:
Yep, since 1.55.